aten/src/ATen/native/TensorAdvancedIndexing.cpp

```diff
@@ -590,7 +591,15 @@ static Tensor & masked_fill_impl_cpu(Tensor & self, const Tensor & mask, Scalar
 Tensor & masked_fill__cpu(Tensor& self, const Tensor & mask, Scalar value) {
```

Comment:
You don't need to sepate cuda and cpu any more here, please combine.

Reply:
Sure.